src/App/Fossa/Config/ConfigFile.hs

```diff
@@ -182,9 +181,6 @@ mergeFileCmdMetadata meta cfgFile =
         , projectReleaseGroup = projectReleaseGroup meta <|> (configProject cfgFile >>= configReleaseGroup)
         }
 
-empty :: ConfigFile
```

Comment:
This is never used, and I keep having to update it when I add something to `ConfigFile`, so I just deleted it